create_bringup/CMakeLists.txt

```diff
@@ -1,11 +1,13 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(create_bringup)
 
-find_package(catkin REQUIRED)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
There's no C++ code in this package. I guess we can remove this line.

Reply:
Removed.